packages/message-handler/src/abstract-message-handler.ts

```diff
@@ -18,9 +18,7 @@ export abstract class AbstractMessageHandler {
   }
 
   public async handle(message: Message, context: IAgentContext<{}>): Promise<Message> {
-    if (this.nextMessageHandler) {
-      return this.nextMessageHandler.handle(message, context)
-    }
+    if (this.nextMessageHandler) return this.nextMessageHandler.handle(message, context)
```

Comment:
Please revert this
```suggestion
    if (this.nextMessageHandler) {
      return this.nextMessageHandler.handle(message, context)
    }
```

Reply:
We can revert this, but we have a suggestion.

The project is currently setup to use `prettier`:
```json
    "prettier": "prettier --write \"{packages,docs,__tests__,!build}/**/*.{ts,js,json,md,yml}\"",
```
And `prettier` is opinionated about single lines inside an `if` block:
![image](https://github.com/decentralized-identity/veramo/assets/114397409/74b616f8-9305-4e76-8830-60d07cd313ac)

> [!Note]
> This script has not been required for PRs and many files are currently violating the `prettier` formatting rules. You could add this to your CI to automatically catch when contributors fail to run `pnpm prettier` before submission.

Reply:
I agree. We have to re-enable automatic formatting.
We've been putting it off because of pending PRs that would show conflicts, but you are 100% right it should be done.